crun.1

```diff
@@ -1,3 +1,4 @@
+'\" t
```

Comment:
I used `github.com/cpuguy83/go-md2man/v2@v2.0.4` to generate that, do we have another preferred version?

Reply:
I was just using the version available on Fedora, using the latest version is better